p4-16/spec/P4-16-spec.mdk

```diff
@@ -5335,8 +5325,9 @@ following behavior after the expression evaluation is interrupted.
 ## Conditional statement { #sec-cond-stmt }
 
 The conditional statement uses standard syntax and semantics familiar from many programming languages.
+
 However, the condition expression in P4 is required to be a Boolean (and not
-an integer).
+an integ
```

Comment:
at least this one looks like a bug

Reply:
PR: [https://github.com/p4lang/p4-spec/pull/1098](https://github.com/p4lang/p4-spec/pull/1098)